test/WebJobs.Script.Tests/Diagnostics/OpenTelemetry/OpenTelemetryConfigurationExtensionsTests.cs

```diff
@@ -235,87 +231,6 @@ public void ResourceDetectorLocalDevelopment()
             Assert.Equal(3, resource.Attributes.Count());
         }
 
-        [Fact]
-        public void ConfigureTelemetry_Should_UseOpenTelemetryWhenModeSetAndAppInsightsAuthStringClientIdPresent()
```

Comment:
How much test coverage are we losing here? Do we have tests to verify how telemetry mode impacts the service collection?

Reply:
We’re only losing the credential‑related test cases. Other tests remain in place, including those that look for telemetry client if in default/AppInsights SDK mode or, in the case of OTEL, a Trace provider.